build/dappnode/scripts/dappnode_install_pre.sh

```diff
@@ -124,6 +124,28 @@ install_docker_compose() {
     fi
 }
 
+
+install_wireguard_dkms() {
+    ##############################################
+    ##############################################
+    ####        WIREGUARD INSTALLATION        ####
+    ##############################################
+    ##############################################
+    apt-get update -y
+    if [ -f "/etc/os-release" ] && grep -q "buster" "/etc/os-release"; then
+        echo "deb http://deb.debian.org/debian/ buster-backports main" > /etc/apt/sources.list.d/buster-backports.list
+        printf 'Package: *\nPin: release a=buster-backports\nPin-Priority: 90\n' > /etc/apt/preferences.d/limit-backports
+    fi
+
+    apt-get install wireguard-dkms -y | tee -a $LOG_FILE
+
+    if  modprobe wireguard >/dev/null 2>&1 ; then
+        echo -e "\e[32m \n\n Verified wiregurd-dkms installation \n\n \e[0m" 2>&1 | tee -a $LOG_FILE
+    else
```

Comment:
should we install DNP_VPN in this case?

Reply:
Not here because this installs underlying system, DNP_VPN is DAppNode package which gets installed in install script via docker.